baitap-19-10-2020/bai-3.js

```diff
@@ -0,0 +1,12 @@
+const compare = (a) => {

```

Comment:
compare cái gì ???
không đặt tên biến tối nghĩa như `a, b, ....`

```suggestion
const compareVS100 = number => {

```